src/constants.ts

```diff
@@ -96,7 +116,10 @@ export const localhost: string = '127.0.0.1';
 export const tsDefaultOutDir: string = 'dist';
 export const tsConfigFileName: string = 'tsconfig.json';
 
-export const localEmulatorConnectionString: string = 'UseDevelopmentStorage=true';
+export const localEventHubsEmulatorConnectionStringDefault: string = 'MemoryF';
+export const localEventHubsEmulatorConnectionStringAlternate: string = 'Memory';
+export const localStorageEmulatorConnectionString: string = 'UseDevelopmentStorage=true';
+export const localEventHubsEmulatorConnectionRegExp: RegExp = new RegExp(`${localEventHubsEmulatorConnectionStringDefault}|${localEventHubsEmulatorConnectionStringAlternate}`);
```

Comment:
Does case-sensitivity matter at all here?

Reply:
I remember testing two of those cases and case-sensitivity seemed to matter

Reply:
Then I think it'd be good to add the `i` flag just in case.

Reply:
I think you read that as case-insensitivity then 🙂

Reply:
I think you might have read that backwards? :)